File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/csharp/valid.cs
// Fixture — café 日本語 before declarations
using System;

namespace Fixture
{
    /// <summary>A greeter.</summary>
    public class Greeter
    {
        /// <summary>Say hello.</summary>
        public string Hello(string name) => "hi " + name;
        public Greeter() {}
    }

    public interface IShape { double Area(); }
    public struct Point { public int X; }
    public enum Color { Red, Blue }
    public record Pair(int A, int B);
}

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/java/valid.java
// Fixture — café 日本語 before declarations
package fixture;

import java.util.List;

/** A greeter. */
public class Greeter {
    private int count;

    /** Say hello. */
    public String hello(String name) {
        return "hi " + name;
    }

    public Greeter() {}
}

interface Shape {
    double area();
}

enum Color { RED, BLUE }

record Pair(int a, int b) {}
